code/controllers/subsystem/statpanel.dm

```diff
@@ -395,11 +395,62 @@ SUBSYSTEM_DEF(statpanels)
 	listed_turf = new_turf
 	if(!client)
 		return
-	if(!client.obj_window)
-		client.obj_window = new(client)
-	if(listed_turf)
-		client.stat_panel.send_message("create_listedturf", listed_turf.name)
-		client.obj_window.start_turf_tracking()
-	else
-		client.stat_panel.send_message("remove_listedturf")
-		client.obj_window.stop_turf_tracking()
+
+	var/datum/tgui/ui = SStgui.try_update_ui(usr, src, "LootPanel")
+	if(!ui)
+		ui = new(usr, src, "LootPanel")
+		ui.open()
+
+/mob/ui_data(mob/user)
+	var/list/data = list()
+
+	var/list/contents = list()
+	for(var/atom/thing in listed_turf.contents)
+		var/generated_string
+		if(ismob(thing) || length(thing.overlays) > 2)
+			generated_string = costly_icon2html(thing, client, sourceonly=TRUE)
+		else
+			generated_string = icon2html(thing, client, sourceonly=TRUE)
```

Comment:
Is this being called on autoupdate? :(

Reply:
I was thinking about just making this static ui data and not updating at all. It takes like two seconds to reopen, very similar to orbit where it's so expensive that I'd want players to have to manually do it
(And, tbh, very lazy way of making it work)

Reply:
Yeah that is pretty terrible :(